deviceModeInit.js

```diff
@@ -86,16 +86,11 @@ var rudderTracking = (function () {
     fetchCart()
       .then((cart) => {
         const needToUpdateCart = checkCartNeedsToBeUpdated(cart);
-        const needToUpdateCookie = checkCookieNeedsToBeUpdated();
```

Comment:
Why are you removing this? `needToUpdateCart` will be true only once a new cart has been created for the entire session.

Reply:
Yes, and only inn that  case only we are calling `sendRudderIdentitiferEventPeriodically` which will keep sending rudderIdentifier at a periodically time interval so there is no need of cookies. Hence no need to check those.

Reply:
But when we refresh the page does it will be false and will then the periodic calls happen?

Reply:
No , Incorporated the cookie now